app/src/main/java/is/xyz/mpv/TouchGestures.kt

```diff
@@ -114,6 +114,6 @@ class TouchGestures(val width: Float, val height: Float, val observer: TouchGest
             }
         }
 
-        return true
+        return false
```

Comment:
doesn't `false` mean the event was not handeled?

Reply:
You're right. I erroneously thought this return statement is reached only if no gesture has been handled.
I'll fix this.

Reply:
There are two options:

- Remove onKeyDown from MPVActivity and handle tap with TouchGestures as a PropertyChange;
- Return false and leave the tap handler in MPVActivity.

What do you think is better?

Reply:
latter seems much cleaner to me